gapic/src/main/com/google/gapid/widgets/Widgets.java

```diff
@@ -453,6 +456,23 @@ public static Text createTextarea(Composite parent, String text) {
     return result;
   }
 
+  public static StyledText createStyledText(Composite parent, String text, boolean editable) {
```

Comment:
I don't think editable is needed since there is never going to be a use case where we allow the user to edit these values. We can just silently do setEditable(false) everywhere. If at all we hit some feature where we allow the user to edit these things, we can enable then.